css/design_system/components/card.scss

```diff
@@ -0,0 +1,111 @@
+ /**
+ * @title Basic Card
+ * @category components
+ * @component_type card
+ * @value s-card
+ * @description The basic card allows for a title, subtitle, content, and footer - all of which are optional. The following example shows a card that uses all options.
+ *
+ * @example
+ * <div class="s-card">
+ *  <div class="s-card-header">
+ *    <h2 class="s-card-title">Card Title</h2>
+ *    <h3 class="s-card-subtitle">Card Subtitle</h3>
+ *  </div> 
+ *  <div class="s-card-content">
+ *    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum risus odio quis sit mattis in luctus. Dignissim auctor aliquam suspendisse non in ac, purus. Aliquam arcu justo quam lorem at id proin morbi. A arcu accumsan, eu id fringilla non.</p>
+ *  </div>
+ *  <div class="s-card-footer">
+ *    <p>Footer content</p>
+ *  </div>
+ * </div>
+ */
+
+.s-card {
+  @extend .s-p-4;
+  @extend .s-shadow-sm;
+  border-radius: var(--s-space-1);
+  border-top: 8px solid var(--s-color-cyan-600);
+
+  .s-card-header {
+    .s-card-title {
+      @extend .s-text-gray-800;
+      @extend .s-text-uppercase;
+      @extend .s-m-0;
+      @extend .s-font-heavy;
+      font-size: var(--s-font-size-lg); 
+    }
+
+    .s-card-subtitle {
+      @extend .s-text-gray-600;
+      margin-top: var(--s-space-1);
+      letter-spacing: 0.75px;
+      font-size: var(--s-font-size-md);
+    }
+  }
+
+  .s-card-content * {
+    @extend .s-text-gray-800;
+  }
+
+  .s-card-footer {
+    @extend .s-text-gray-800;
+    @extend .s-text-body-xs;
+    @extend .s-text-end;
+
+    p {
+      @extend .s-m-0;
+    }
```

Comment:
We can probably get rid of this style and let people apply their own classes to elements inside of a footer.
```suggestion
```